template/en/default/mfa/duo/verify.html.tmpl

```diff
@@ -57,7 +57,7 @@ $(function() {
 
 <p>
   <b>[% reason FILTER html %]</b> requires verification.<br>
-  [% UNLESS is_enrolment %]
+  [% UNLESS is_enrollment %]
```

Comment:
This has also been broken, because `is_enrollment` is defined at the top of the template.

Reply:
This mismatch is due to #232.